src/components/Credits/index.tsx

```diff
@@ -2,8 +2,7 @@ import React, { FC } from 'react';
 import styled from 'styled-components';
 import useLocalizedContent from '../../utils/hooks/useLocalizedContent';
 
-const logoCitylab =
-  'https://logos.citylab-berlin.org/logo-citylab-berlin-outline.svg';
+const logoCitylab = 'https://logos.citylab-berlin.org/logo-citylab-color.svg';
```

Comment:
Why are we linking to external sites for logos?
Also shouldn't logos not be part if content?

Reply:
valid point. I guess in the past the idea was to link it from a single source of truth so updating logos becomes easier. But I agree we should discuss how to handle this (as for example, an accidental change in the source wrongly changed the logos on many websites).